internal/quickfort/build.lua

```diff
@@ -766,9 +771,11 @@ function do_run(zlevel, grid)
 end
 
 function do_orders(zlevel, grid)
-    local stats = nil
-    print('"quickfort orders" not yet implemented for mode: build')
-    return stats
+    local stats = {invalid_keys={label='Invalid key sequences', value=0}}
+    local buildings = {}
+    stats.invalid_keys.value = quickfort_building.init_buildings(
+            zlevel, grid, buildings, building_db, building_aliases, true)
+    quickfort_orders.enqueue_orders(stats, buildings, building_db)
```

Comment:
This call needs a `ctx` argument. It looks like you already addressed this in #185 in a non-trivial way, so I'm going to close this PR in favor of that one.